Makefile

```diff
@@ -28,7 +28,6 @@ db: db_sqlite
 	usql pg://postgres:pgpass@localhost:55432/testdb?sslmode=disable -f testdata/ddl/postgres95.sql
 	usql pg://postgres:pgpass@localhost:55413/testdb?sslmode=disable -f testdata/ddl/postgres.sql
 	usql my://root:mypass@localhost:33306/testdb -f testdata/ddl/mysql56.sql
-	usql my://root:mypass@localhost:33308/testdb -f testdata/ddl/mysql.sql
```

Comment:
Not sure how to execute such script with [stored procedure and function](https://github.com/k1LoW/tbls/pull/341/files#diff-17bc6cdcfcdef7fa8e80f0062cbccdda0b66f2fbcfa8234f0712934fc143b0f1R108) via `usql`. So it is moved to be executed during [container creation](https://github.com/k1LoW/tbls/pull/341/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R40)

Reply:
Does this mean that CREATE PROCEDURE could not be executed using `usql` ?

Reply:
I wasn't able to execute this file with the following code using `usql`
```
DELIMITER //
CREATE PROCEDURE GetAllComments()
BEGIN
	SELECT * FROM comments;
END//
DELIMITER ;
```